src/aihawk_easy_applier.py

```diff
@@ -434,7 +434,11 @@ def _handle_upload_fields(self, element: WebElement, job) -> None:
                     self._create_and_upload_resume(element, job)
             elif 'cover' in output:
                 logger.debug("Uploading cover letter")
-                self._create_and_upload_cover_letter(element, job)
+                if job.cover_letter_path is not None and os.path.exists(job.cover_letter_path):
```

Comment:
move this check to method something like  if_coverletter_exits or something silimar. method name should be check objective.

Reply:
I guess you are missing case where cover is in plain text resume.yaml